databricks/koalas/indexes.py

```diff
@@ -2271,6 +2271,40 @@ def view(self) -> "Index":
         """
         return self.copy()
 
+    def to_list(self) -> List:
+        """
+        Return a list of the values.
+
+        These are each a scalar type, which is a Python scalar
+        (for str, int, float) or a pandas scalar
+        (for Timestamp/Timedelta/Interval/Period)
+
+        .. note:: This method should only be used if the resulting list is expected
+            to be small, as all the data is loaded into the driver's memory.
+
+        Examples
+        --------
+        Index
+
+        >>> idx = ks.Index([1, 2, 3, 4, 5])
+        >>> idx.to_list()
+        [1, 2, 3, 4, 5]
+
+        MultiIndex
+
+        >>> tuples = [(1, 'red'), (1, 'blue'), (2, 'red'), (2, 'green')]
+        >>> midx = ks.MultiIndex.from_tuples(tuples)
+        >>> midx.to_list()
+        [(1, 'red'), (1, 'blue'), (2, 'red'), (2, 'green')]
+        """
+        # pandas<0.24 doesn't support `to_list()` as an alias for `tolist()`
+        if LooseVersion(pd.__version__) < LooseVersion("0.24"):
+            return self.to_pandas().tolist()
```

Comment:
Why don't you just call `tolist` in all cases?

Reply:
Sounds good. Then I'll address `Series.to_list()`, too. I just referred to that implementation.